sampleWorkspace/localSchema/apollo.config.ts

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  client: {
+    service: {
+      name: "localSchema",
+      localSchemaFile: "./starwarsSchema.graphql",
+    },
+  },
+};
```

Comment:
We need one "TypeScript, but CommonJS" config for testing, and the rover config seemed like a better candidate for yaml anyways